docs/architecture/cloud-native/map-eshoponcontainers-azure-services.md

```diff
@@ -1,13 +1,69 @@
+
+![eShopOnContainers Architecture](media/eshoponcontainers-architecture.png)
+**Figure 3-X**. The eShopOnContainers Architecture.
+
+The server-side components of this architecture all map easily to Azure services.
+
+## Container orchestration and clustering
+
+The application's container-hosted services, from ASP.NET Core MVC apps to individual Catalog and Ordering microservices, can all be hosted and managed in Azure Kubernetes Service (AKS). The application can run locally on Docker and Kubernetes, and the same containers can then be deployed to staging and production environments hosted in AKS. This process can be completely automated as we'll see in the next section.
+
+AKS provides management services for individual clusters of containers. The application will deploy separate AKS clusters for each microservice shown in the architecture diagram above. This approach allows each individual service to each independently according to its resource demands. Each microservice can also be deployed independently, and ideally such deployments should incur zero system downtime.
+
+## API Gateway
+
+The eShopOnContainers application has multiple front end clients and multiple different backend services. There is no one-to-one correspondence between the client applications and the microservices that support them. In such a scenario, there may be a great deal of complexity when writing client software to interface with the various backend services in a secure manner. Each client would need to address this complexity on its own, resulting in duplication and many places in which to make updates as services change or new policies are implemented.
+
+Azure API Management (APIM) helps organizations publish APIs in a consistent, manageable fashion. APIM consists of three components: the API Gateway, and administration portal (the Azure portal), and a developer portal.
+
+The API Gateway accepts API calls and routes them to the appropriate backend API. It can also provide additional services like verification of API keys or JWT tokens and API transformation on the fly without code modifications (for instance, to accommodate clients expecting an older interface).
+
+The Azure portal is where you define the API schema and package different APIs into products. You also configure user access, view reports, and configure policies for quotas or transformations.
+
+The developer portal serves as the main resource for developers. It provides developers with API documentation, an interactive test console, and reports on their own usage. Developers also use the portal to create and manage their own accounts, including subscription and API key support.
+
+Using APIM, the eShopOnContainers application exposes several different products of services, each providing a backend for a particular front end client.
```

Comment:
eShop does not use API Management. For simplicity, we use Ocelot in the implementation but do recommend to use API Management for more complex scenarios. See this: https://docs.microsoft.com/en-us/dotnet/architecture/microservices/architect-microservice-container-applications/direct-client-to-microservice-communication-versus-the-api-gateway-pattern#azure-api-management 
Can you rephrase so it doesn't contradict?